apps/cevent_frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styles from '@/styles/components/Footer.module.css';
+
+const Footer: React.FC = () => {
+  return (
+    <div className={styles.footer}>
+      <div>
+        <h1>Cvent</h1>
+        <h2>
+          Descripción sobre qué es Cvent, Descripción sobre qué es Cvent,
```

Comment:
Replace the placeholder text, for real text. Like, instead of "Descripción sobre qué es Cvent", write a real description of it. So, that we do not have to create a new user story for doing that. Thx.